test/utils/helpFn.ts

```diff
@@ -116,6 +122,31 @@ export const getKeyringPair = async () => {
   return keyringPair;
 }
 
+/**
+ * Get keyring pair for moonbeam testing
+ * @returns keyring pair
+ */
+export const getMoonbeamKeyringPair = async (): Promise<KeyringPair> => {
+  await waitReady();
+  const keyringECDSA = new Keyring({ type: 'ethereum' });
+  let moonbeamKeyringPair = null;
+  if (getEnv() === 'Turing Dev' && _.isEmpty(process.env.MNEMONIC)) {
+    moonbeamKeyringPair = keyringECDSA.addFromSeed(hexToU8a(ALITH_PRIVATE_KEY), undefined, 'ethereum');
+  } else {
+    if (_.isEmpty(process.env.MNEMONIC)) {
+      throw new Error('The MNEMONIC environment variable is not set.')
+    }
```

Comment:
These lines 133 - 138 don’t look like formatted by eslint. Did you run `eslint --fix` against all files?

Reply:
Yes, these code snippets have been formatted using ESLint.